AdminNotificationManagerPlugin.inc.php

```diff
@@ -146,19 +148,31 @@ function getTemplatePath($inCore = false) {
 	 * @return array
 	 */
 	function _getAdminList() {
-		$user = null;
+		$arrayOfContexts = $this->_getContexts();
 		$users = array();
+		$userId = array();
+		$user = null;
```

Comment:
Here `$user = null` is still marked as a deletion and addition.  If it were left just below `function _getAdminList() {` or just above `$users = array();`, then it would be marked as unchanged.

Reply:
I changed the order of my variables being declared, which may have resulted in this as both a deletion and addition, as I was changing the position of it. I don't believe it would cause any functional problems

Reply:
Not a big deal, but `$user = null` still shows up as a delete and addition.